IDE/STARCORE/.gitignore

```diff
@@ -0,0 +1 @@
+.metadata/.plugins/org.eclipse.core.resources/.history
```

Comment:
Is the `.metadata` folder really required?

Reply:
Yes, the metadata contains all the rules for locating the source and headers along with project configuration settings etc.